samples/otlptracegrpc/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "sample-app-grpc",
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "main": "index.ts",
+  "scripts": {
+    "start": "node ./app.ts"
+  },
```

Comment:
Can you copy in the lint, clean, and fix scripts from here? https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/c24774bc6c79df53d257003cb2facc4fb8bd9052/packages/opentelemetry-cloud-trace-exporter/package.json#L8-L18

Then you will need to include the `gts` dev dependency https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/c24774bc6c79df53d257003cb2facc4fb8bd9052/packages/opentelemetry-cloud-trace-exporter/package.json#L52

Reply:
Updated scripts.